src/state/farms/getFarmsPrices.ts

```diff
@@ -72,8 +72,8 @@ const getFarmQuoteTokenPrice = (farm: Farm, quoteTokenFarm: Farm, bnbPriceBusd:
   return BIG_ZERO
 }
 
-const fetchFarmsPrices = async (farms) => {
-  const bnbBusdFarm = farms.find((farm: Farm) => farm.pid === 252)
+const getFarmsPrices = (farms: Farm[]) => {
```

Comment:
@ChefHutch I renamed this function because it is not really fetching anything nor does it need to be asynchronous.